components/infobox/wikis/arenaofvalor/infobox_team_custom.lua

```diff
@@ -51,4 +74,24 @@ function CustomTeam:addToLpdb(lpdbData, args)
 	return lpdbData
 end
 
+function CustomTeam:_createRegion(region)
+	if String.isEmpty(region) then
+		return
+	end
+
+	local regionData = Region.run({region = region})
+	if Table.isEmpty(regionData) then
+		return
+	end
+
+	local remappedRegion = REGION_REMAPPINGS[regionData.region:lower()]
+	if remappedRegion then
+		return CustomTeam:_createRegion(remappedRegion)
+	end
+
+	Variables.varDefine('region', regionData.region)
```

Comment:
shouldn't the vardefine be in the according function where other wiki vars are set?

Reply:
yeah but i think there are places relying on americas == usca flag

Reply:
alright